src/app/cli/src/graphql.ml

```diff
@@ -84,6 +84,15 @@ module Types = struct
     (** Unix form of time, which is the number of milliseconds that elapsed from January 1, 1970 *)
     let date = Time.to_string
 
+    (** string representation of JSON version of trust status *)
+    let trust_status st =
+      Trust_system.Peer_status.to_yojson st |> Yojson.Safe.to_string
```

Comment:
Can you model this as graphql types rather than stringifying JSON?

Proposal for model:

```graphql
type BannedStatus {
   bannedUntil: String # a timestamp, null means Unbanned
}

type TrustStatus {
   trust: Float!
   banned: BannedStatus!
}
```

Then the all can take a `non_null (list (non_null trust_status))`

Reply:
OK

Reply:
except that the `all` returns a list of IP address, status pairs